docs/tutorials/get-started/servo-mousemover.md

```diff
@@ -188,7 +190,8 @@ Create a [servo component](/components/servo/):
 
 1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
 1. Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
-1. Select the `servo` type, then select the `pi` model.
+1. Select the `servo` type, then select the `viam:raspberry-pi:pi-servo` model if you are using a Raspberry Pi 3, 4 or Zero 2 W.
+   If you are using a Raspberry Pi 5, use the `gpio` model.
```

Comment:
```suggestion
1. Select the `servo` type, then select the `viam:raspberry-pi:pi-servo` model.
```

Reply:
apologies, pi-servo should work with any board.

Reply:
awesome thanks!